Facepunch.Steamworks/Client.cs

```diff
@@ -114,7 +114,8 @@ public Client( uint appId )
             SteamId = native.user.GetSteamID();
             BetaName = native.apps.GetCurrentBetaName();
             OwnerSteamId = native.apps.GetAppOwner();
-            InstallFolder = new DirectoryInfo( native.apps.GetAppInstallDir( AppId ) );
+            if (!String.IsNullOrEmpty(native.apps.GetAppInstallDir(AppId)) && Directory.Exists(native.apps.GetAppInstallDir(AppId)))
```

Comment:
Could probably do with caching native.apps.GetAppInstallDir(AppId) to a temporary var instead of calling it twice